internal/pkg/service/stream/storage/model/repository/slice/slice_rotate_test.go

```diff
@@ -110,3 +111,74 @@ func TestSliceRepository_Rotate(t *testing.T) {
 	// -----------------------------------------------------------------------------------------------------------------
 	etcdhelper.AssertKVsFromFile(t, client, "fixtures/slice_rotate_snapshot_001.txt", etcdhelper.WithIgnoredKeyPattern("^definition/|storage/file/|storage/slice/all/|storage/stats/|storage/secret/|storage/volume"))
 }
+
+func TestSliceRepository_Rotate_WithClosingFile(t *testing.T) {
```

Comment:
There is also missing test, where we change from `stats.Staging` -> `stats.Total` statistics in filerotation module. Could you cover that case as well please :slightly_smiling_face:

Reply:
Already covered here: https://github.com/keboola/keboola-as-code/commit/4ae73a42bd3a293886258d9c7ee070c0ceda7f67